### ISSUE TITLE
Deepin 15.7 ... first impressions ... unstable flickering mess currently !!!

### ISSUE THREAD

**Issue by aiamuzz** [linuxdeepin/developer-center#595]
I am just uploading the screen recorder in GIF format for uploading it here ... If you follow the mouse cursor and the clicks you'll know how erratic the latest 15.7 update is ...

![deepin 15 7](https://user-images.githubusercontent.com/3283676/44455891-d7865400-a61c-11e8-9b70-74b470dbfcea.gif)

- 

1.  Unable to access the dock(smart hide) ... as you can see its all flickery ... only way i can access the launcher is with the corner settings.

2. Not able to go back to full screen launcher as it doesn't respond as intended when clicked on the full screen button.

3. Control Center is flickery not easy to access or navigate option, only accessible through corner settings as the dock isn't accessible. it keeps disappearing unless i click on the empty desktop and then go back to the corner.

4. My 10+ disk partitions aren't auto mounting on startup, a couple of random partition mount ... i have currently mounted the rest of them manually.

5. as you can see in the screen recorder GIF ... the keyboard shortcut's both system and custom shortcuts aren't showing ... back in 15.6 only the custom shortcuts weren't populating but now all shortcuts have disappeared from the control center and none of the shorcuts are working as they should.
;
;
;
;
;
I would post more issues, if only i could test them if the system was stable enough ...

PS : I upgraded my system through `sudo apt-get dist-upgrade` as the control center method wasn't really responding.

**Comment by es02** [comment]
+1
Manjaro-Deepin on a Dell Precision M6700 and it's barely usable.

- Dock is constantly restarting, some dock widgets like wifi don't load at all.
- Applications menu is super unresponsive to opening commands including by keyboard.
- login screen is white with background and some UI elements not rendering
- System Information does not populate

turning off window effect restored  tiny amount of sanity however this is still unusable

**Comment by hundsboog** [comment]
I tested two versions, Deepin 15.7 directly and Manjaro Deepin 15.7 version. Both show the behaviour as shown above. It looks like two docks or icon sets are fighting to be shown.

**Comment by es02** [comment]
I should add that whole my dock will sometimes settle down (missing power, audio and networking applets) clicking on anything on that dock will usually re-trigger the bug behavior

**Comment by felixonmars** [comment]
I am only the Arch packages' maintainer here and I cannot reproduce any of these on Arch :(

@hualet Any ideas?

**Comment by hualet** [comment]
It must be dde-session-daemon's problem, journalctl -b 0 /usr/lib/deepin-daemon/dde-session-daemon may give us some clue. @aiamuzz

**Comment by aiamuzz** [comment]
@hualet ...
attached log file ...
[daemon_log.txt](https://github.com/linuxdeepin/developer-center/files/2321424/daemon_log.txt)

**Comment by hualet** [comment]
@electricface

**Comment by aiamuzz** [comment]
@electricface  ... @hualet  ...

not a livable condition currently ... i have installed more than a couple of updates you've pushed, hoping this instability would be fixed ... nothing so far ... life's really difficult currently ... i am using the system by keeping a file manager and terminal window open all through out the session ... also i am keeping applications that minimize to tray when closed by minimizing them rather than closing them bcos the system tray on the dock is missing and the overall dock is un responsive ... open and minimized applications with Alt+tab are the only way i can get through my daily task on this highly unstable system.

fix badly needed !!!

"Enjoy the better Performance" is more of a misnomer currently ... quite the opposite in fact worse than normal ...

Have you put a hold on the upgrade currently ?

Besides the official testing phase, which clearly seem to have failed to catch the issues ...

Hope deepin has concurrent system of upgrade, where only a fraction of users are pushed updates and depending on the stability of the upgraded ... more and more users are subsequently pushed ... covering the entire user base ... at least this way the entire user base can be protected from a bad or unstable upgrade. A system of Incremental phase by phase upgrade system ...

**Comment by SeriousM** [comment]
I have the same problem, the taskbar is flickering and freezing on my ThinkPad T460p (Intel Core i7).
Even the keyboard commands like win-key for launcher is not working.
Clicking on the taskbar is make it even worse.
I'm not a linux dev so I don't really know where the problem is but could someone give a estimation till the problem gets a patch?
Thanks

**Comment by electricface** [comment]
According to the log, your dde-session-daemon program is constantly restarting, but the reason for the problem is not found in the log. Please execute the command in the terminal: "killall dde-session-daemon; /usr/lib/deepin-daemon/dde-session-daemon" and send the output.

**Comment by aiamuzz** [comment]
the output of the command is as follows ...

```
AH15:~$ killall dde-session-daemon; /usr/lib/deepin-daemon/dde-session-daemon >>dde-session-daemon-restart.log
dde-session-daemon: no process found
panic: you should call *proxy.Object.InitSignalExt() first

goroutine 75 [running]:
pkg.deepin.io/lib/dbusutil/proxy.(*Object).checkSignalExt(0xc4200a5740)
	/usr/share/gocode/src/pkg.deepin.io/lib/dbusutil/proxy/object.go:164 +0xcd
pkg.deepin.io/lib/dbusutil/proxy.(*Object).initPropertiesChangedHandler(0xc4200a5740, 0xec8718, 0xc4200a5758)
	/usr/share/gocode/src/pkg.deepin.io/lib/dbusutil/proxy/object.go:120 +0x40
pkg.deepin.io/lib/dbusutil/proxy.(*Object).ConnectPropertyChanged_(0xc4200a5740, 0xea1115, 0x23, 0xe8b4ba, 0x12, 0xc420423970, 0x0, 0x0)
	/usr/share/gocode/src/pkg.deepin.io/lib/dbusutil/proxy/object.go:148 +0x91
pkg.deepin.io/lib/dbusutil/proxy.PropUint32.ConnectChanged(0xf37ea0, 0xc4200a5740, 0xe8b4ba, 0x12, 0xc4200a58c0, 0x2a, 0x1)
	/usr/share/gocode/src/pkg.deepin.io/lib/dbusutil/proxy/prop.go:284 +0xd2
pkg.deepin.io/dde/daemon/network.(*Manager).newDevice(0xc420434120, 0xc420949200, 0x29, 0x0, 0xf345a0, 0xc42001f0f0)
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/manager_device.go:215 +0x100e
pkg.deepin.io/dde/daemon/network.(*Manager).addDevice(0xc420434120, 0xc420949200, 0x29)
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/manager_device.go:312 +0xd8
pkg.deepin.io/dde/daemon/network.(*Manager).initDeviceManage(0xc420434120)
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/manager_device.go:89 +0x197
pkg.deepin.io/dde/daemon/network.(*Manager).init(0xc420434120)
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/manager.go:196 +0x2b1
pkg.deepin.io/dde/daemon/network.(*Daemon).Start.func1()
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/main.go:91 +0x2d
created by pkg.deepin.io/dde/daemon/network.(*Daemon).Start
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/main.go:90 +0x4f6

```

**Comment by aiamuzz** [comment]
ok ... here you go ...

```
dde-session-daemon: no process found
<info> main.go:121: App log level: 5
<info> loader.go:151: enable module dock
<info> app_entry.go:221: attach window id: 33554435, wmClass: "Deepin-terminal" "deepin-terminal", wmState: [477], wmWindowType: [403], wmAllowedActions: [529 530 538 539 533 534 535 536 543 544], hasXEmbedInfo: false, hasWmTransientFor: false
<info> app_entry.go:221: attach window id: 81788929, wmClass: "Google-chrome" "google-chrome", wmState: [381 382], wmWindowType: [403], wmAllowedActions: [529 530 538 539 533 534 535 536 543 544], hasXEmbedInfo: false, hasWmTransientFor: false
<info> app_entry.go:221: attach window id: 81788939, wmClass: "Google-chrome" "google-chrome", wmState: [381 382], wmWindowType: [403], wmAllowedActions: [529 530 538 539 533 534 535 536 543 544], hasXEmbedInfo: false, hasWmTransientFor: false
<info> app_entry.go:221: attach window id: 81788940, wmClass: "Google-chrome" "google-chrome", wmState: [381 382], wmWindowType: [403], wmAllowedActions: [529 530 538 539 533 534 535 536 543 544], hasXEmbedInfo: false, hasWmTransientFor: false
<info> loader.go:159: enable module dock done, cost 26.836938ms
<info> loader.go:151: enable module trayicon
<info> loader.go:159: enable module trayicon done, cost 7.620566ms
<info> loader.go:151: enable module x-event-monitor
<info> traymanager.go:369: Add tray icon 71303704 name: "NetworkManager Applet"
<info> loader.go:159: enable module x-event-monitor done, cost 960.502µs
<info> loader.go:151: enable module network
<warning> proxychains.go:106: config is invalid
<info> loader.go:159: enable module network done, cost 1.839776ms
<info> loader.go:151: enable module audio
<info> manager.go:168: initialize network
<info> manager_config.go:70: config file: /home/muz/.config/deepin/network.json
<info> loader.go:159: enable module audio done, cost 12.091001ms
<info> loader.go:151: enable module screensaver
<info> loader.go:159: enable module screensaver done, cost 2.902916ms
<info> loader.go:151: enable module sessionwatcher
<info> loader.go:159: enable module sessionwatcher done, cost 7.060052ms
<info> loader.go:151: enable module power
<info> manager.go:119: LidIsPresent true
<info> loader.go:159: enable module power done, cost 7.025651ms
<info> loader.go:151: enable module launcher
<info> submodule.go:58: submodule PowerSavePlan start
<info> power_save_plan.go:175: cancel all tasks
<info> power_save_plan.go:177: all tasks done!
<info> screensaver.go:205: SetTimeout to  900 0 false
<info> submodule.go:58: submodule LidSwitchHandler start
<info> loader.go:159: enable module launcher done, cost 97.974084ms
<info> loader.go:151: enable module service-trigger
<info> loader.go:159: enable module service-trigger done, cost 8.284523ms
<info> loader.go:151: enable module clipboard
<info> loader.go:159: enable module clipboard done, cost 56.812µs
<info> loader.go:151: enable module keybinding
<info> loader.go:159: enable module keybinding done, cost 4.824724ms
<info> loader.go:151: enable module appearance
<info> loader.go:159: enable module appearance done, cost 73.154975ms
<info> loader.go:151: enable module inputdevices
<warning> manager_config.go:227: device config for 2732d3965959886183849497a0d872efe57eab25 not exists
<warning> manager_config.go:227: device config for 2732d3965959886183849497a0d872efe57eab25 not exists
<warning> manager_config.go:227: device config for 00:50:56:C0:00:01 not exists
<warning> manager_config.go:227: device config for 00:50:56:C0:00:01 not exists
<warning> manager_config.go:227: device config for 00:50:56:C0:00:08 not exists
<warning> manager_config.go:227: device config for 00:50:56:C0:00:08 not exists
<warning> manager_config.go:227: device config for 00:50:56:C0:00:01 not exists
<warning> manager_config.go:227: device config for 00:50:56:C0:00:08 not exists
<warning> manager_config.go:227: device config for 2732d3965959886183849497a0d872efe57eab25 not exists
<info> loader.go:159: enable module inputdevices done, cost 165.945763ms
<info> loader.go:151: enable module gesture
<info> loader.go:159: enable module gesture done, cost 6.232377ms
<info> loader.go:151: enable module housekeeping
<info> loader.go:159: enable module housekeeping done, cost 23.976µs
<info> loader.go:151: enable module timedate
<info> manager_device.go:103: ignore virtual network interface which driver is unknown /org/freedesktop/NetworkManager/Devices/1
<info> loader.go:159: enable module timedate done, cost 4.500703ms
<info> loader.go:151: enable module bluetooth
<info> loader.go:159: enable module bluetooth done, cost 1.799845ms
<info> loader.go:151: enable module screenedge
<info> config.go:46: load bluetooth config file: /home/muz/.config/deepin/bluetooth.json
<info> loader.go:159: enable module screenedge done, cost 2.142362ms
<info> loader.go:151: enable module fprintd
<info> loader.go:159: enable module fprintd done, cost 6.873949ms
<info> loader.go:151: enable module mime
<warning> manager.go:85: no such file or directory
<info> loader.go:159: enable module mime done, cost 1.851397ms
<info> loader.go:151: enable module miracast
<info> loader.go:159: enable module miracast done, cost 31.509352ms
<info> loader.go:151: enable module systeminfo
<info> loader.go:159: enable module systeminfo done, cost 3.617623ms
<info> loader.go:151: enable module lastore
<info> loader.go:159: enable module lastore done, cost 8.197664ms
<info> manager_connection.go:291: -----------Try get uuid, src: 3c646f6d-a76d-0f86-ead1-4de5d9d62f04
<info> manager_connection.go:293: -----------Try get uuid, result: 
<info> nm_setting_wired.go:28: new wired connection, id=Wired Connection, uuid=, devPath=/org/freedesktop/NetworkManager/Devices/2
<info> adapter.go:69: AdapterAdded adapter AMyLaptop [E0:06:E6:B5:DB:4D]
<info> manager_device.go:316: add device &network.device{nmDev:(*networkmanager.Device)(0xc420444060), mmDevModem:(*modemmanager1.Modem)(nil), nmDevType:0x1, id:"84:34:97:20:79:FF", udi:"/sys/devices/pci0000:00/0000:00:19.0/net/enp0s25", Path:"/org/freedesktop/NetworkManager/Devices/2", State:0x14, Interface:"enp0s25", ClonedAddress:"84:34:97:20:79:FF", HwAddress:"84:34:97:20:79:FF", Driver:"e1000e", Managed:true, Vendor:"Intel 82579LM Gigabit (Lewisville)", UniqueUuid:"3c646f6d-a76d-0f86-ead1-4de5d9d62f04", UsbDevice:false, ActiveAp:"", SupportHotspot:false, MobileNetworkType:"", MobileSignalQuality:0x0}
<info> manager_device.go:110: ignore invalid device type 16
<info> manager.go:104: Not found fprint device
<info> manager_device.go:316: add device &network.device{nmDev:(*networkmanager.Device)(0xc4203a5140), mmDevModem:(*modemmanager1.Modem)(nil), nmDevType:0x2, id:"60:67:20:7E:14:E4", udi:"/sys/devices/pci0000:00/0000:00:1c.3/0000:24:00.0/net/wlo1", Path:"/org/freedesktop/NetworkManager/Devices/4", State:0x1e, Interface:"wlo1", ClonedAddress:"60:67:20:7E:14:E4", HwAddress:"60:67:20:7E:14:E4", Driver:"iwlwifi", Managed:true, Vendor:"Intel Centrino Advanced-N 6205 [Taylor Peak] (Centrino Advanced-N 6205 AGN)", UniqueUuid:"8a7da294-c2b6-146c-addd-a07a24e11d14", UsbDevice:false, ActiveAp:"/", SupportHotspot:true, MobileNetworkType:"", MobileSignalQuality:0x0}
<info> manager_device.go:103: ignore virtual network interface which driver is unknown /org/freedesktop/NetworkManager/Devices/5
<info> manager_device.go:103: ignore virtual network interface which driver is unknown /org/freedesktop/NetworkManager/Devices/6
panic: you should call *proxy.Object.InitSignalExt() first

goroutine 33 [running]:
pkg.deepin.io/lib/dbusutil/proxy.(*Object).checkSignalExt(0xc42034d2c0)
	/usr/share/gocode/src/pkg.deepin.io/lib/dbusutil/proxy/object.go:164 +0xcd
pkg.deepin.io/lib/dbusutil/proxy.(*Object).initPropertiesChangedHandler(0xc42034d2c0, 0xec8718, 0xc42034d2d8)
	/usr/share/gocode/src/pkg.deepin.io/lib/dbusutil/proxy/object.go:120 +0x40
pkg.deepin.io/lib/dbusutil/proxy.(*Object).ConnectPropertyChanged_(0xc42034d2c0, 0xea1115, 0x23, 0xe8b4ba, 0x12, 0xc4201adc90, 0x0, 0x0)
	/usr/share/gocode/src/pkg.deepin.io/lib/dbusutil/proxy/object.go:148 +0x91
pkg.deepin.io/lib/dbusutil/proxy.PropUint32.ConnectChanged(0xf37ea0, 0xc42034d2c0, 0xe8b4ba, 0x12, 0xc42034d320, 0x2a, 0x1)
	/usr/share/gocode/src/pkg.deepin.io/lib/dbusutil/proxy/prop.go:284 +0xd2
pkg.deepin.io/dde/daemon/network.(*Manager).newDevice(0xc42040d680, 0xc420886570, 0x29, 0x0, 0xf345a0, 0xc420408880)
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/manager_device.go:215 +0x100e
pkg.deepin.io/dde/daemon/network.(*Manager).addDevice(0xc42040d680, 0xc420886570, 0x29)
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/manager_device.go:312 +0xd8
pkg.deepin.io/dde/daemon/network.(*Manager).initDeviceManage(0xc42040d680)
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/manager_device.go:89 +0x197
pkg.deepin.io/dde/daemon/network.(*Manager).init(0xc42040d680)
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/manager.go:196 +0x2b1
pkg.deepin.io/dde/daemon/network.(*Daemon).Start.func1()
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/main.go:91 +0x2d
created by pkg.deepin.io/dde/daemon/network.(*Daemon).Start
	/build/dde-daemon-3.2.24/gopath/src/pkg.deepin.io/dde/daemon/network/main.go:90 +0x4f6

```

**Comment by electricface** [comment]
Sorry for my fault, this commit fix https://cr.deepin.io/#/c/dde/dde-daemon/+/38055/

**Comment by SeriousM** [comment]
Could you please add a command line for the installation? Or is this going into release soon? Thanks

**Comment by electricface** [comment]
wget https://ci.deepin.io/job/dde-daemon-ci-debian/2938/artifact/dde-daemon_3.2.24%7Egerrit+38055%7E1_amd64.deb
command: sudo dpkg -i ./dde-daemon_3.2.24~gerrit+38055~1_amd64.deb

**Comment by es02** [comment]
`[es02@greyhat ~]$ sudo dpkg -i ./dde-daemon_3.2.24~gerrit+38055~1_amd64.deb
Selecting previously unselected package dde-daemon.
(Reading database ... 0 files and directories currently installed.)
Preparing to unpack .../dde-daemon_3.2.24~gerrit+38055~1_amd64.deb ...
Unpacking dde-daemon (3.2.24~gerrit+38055~1) ...
dpkg: dependency problems prevent configuration of dde-daemon:
 dde-daemon depends on libasound2 (>= 1.0.16); however:
  Package libasound2 is not installed.
 dde-daemon depends on libatk1.0-0 (>= 1.12.4); however:
  Package libatk1.0-0 is not installed.
 dde-daemon depends on libc6 (>= 2.17); however:
  Package libc6 is not installed.
 dde-daemon depends on libcairo-gobject2 (>= 1.10.0); however:
  Package libcairo-gobject2 is not installed.
 dde-daemon depends on libcairo2 (>= 1.2.4); however:
  Package libcairo2 is not installed.
 dde-daemon depends on libfontconfig1 (>= 2.12.6); however:
  Package libfontconfig1 is not installed.
 dde-daemon depends on libfreetype6 (>= 2.2.1); however:
  Package libfreetype6 is not installed.
 dde-daemon depends on libfribidi0 (>= 0.19.2); however:
  Package libfribidi0 is not installed.
 dde-daemon depends on libgdk-pixbuf2.0-0 (>= 2.25.2); however:
  Package libgdk-pixbuf2.0-0 is not installed.
 dde-daemon depends on libglib2.0-0 (>= 2.55.2); however:
  Package libglib2.0-0 is not installed.
 dde-daemon depends on libgtk-3-0 (>= 3.0.0); however:
  Package libgtk-3-0 is not installed.
 dde-daemon depends on libgudev-1.0-0 (>= 212); however:
  Package libgudev-1.0-0 is not installed.
 dde-daemon depends on libinput10 (>= 0.21.0); however:
  Package libinput10 is not installed.
 dde-daemon depends on libnl-3-200 (>= 3.2.7); however:
  Package libnl-3-200 is not installed.
 dde-daemon depends on libnl-genl-3-200 (>= 3.2.7); however:
  Package libnl-genl-3-200 is not installed.
 dde-daemon depends on libpam0g (>= 0.99.7.1); however:
  Package libpam0g is not installed.
 dde-daemon depends on libpango-1.0-0 (>= 1.14.0); however:
  Package libpango-1.0-0 is not installed.
 dde-daemon depends on libpangocairo-1.0-0 (>= 1.14.0); however:
  Package libpangocairo-1.0-0 is not installed.
 dde-daemon depends on libpulse0 (>= 0.99.1); however:
  Package libpulse0 is not installed.
 dde-daemon depends on librsvg2-2 (>= 2.14.4); however:
  Package librsvg2-2 is not installed.
 dde-daemon depends on libudev1 (>= 183); however:
  Package libudev1 is not installed.
 dde-daemon depends on libx11-6 (>= 2:1.2.99.901); however:
  Package libx11-6 is not installed.
 dde-daemon depends on libxcursor1 (>> 1.1.2); however:
  Package libxcursor1 is not installed.
 dde-daemon depends on libxi6 (>= 2:1.2.99.4); however:
  Package libxi6 is not installed.
 dde-daemon depends on upower; however:
  Package upower is not installed.
 dde-daemon depends on network-manager; however:
  Package network-manager is not installed.
 dde-daemon depends on iso-codes; however:
  Package iso-codes is not installed.
 dde-daemon depends on gvfs-bin; however:
  Package gvfs-bin is not installed.
 dde-daemon depends on gnome-keyring; however:
  Package gnome-keyring is not installed.
 dde-daemon depends on deepin-desktop-base; however:
  Package deepin-desktop-base is not installed.
 dde-daemon depends on deepin-sound-theme; however:
  Package deepin-sound-theme is not installed.
 dde-daemon depends on user-setup; however:
  Package user-setup is not installed.
 dde-daemon depends on deepin-metacity; however:
  Package deepin-metacity is not installed.
 dde-daemon depends on deepin-desktop-schemas (>= 3.2.15); however:
  Package deepin-desktop-schemas is not installed.
 dde-daemon depends on mobile-broadband-provider-info; however:
  Package mobile-broadband-provider-info is not installed.
 dde-daemon depends on bamfdaemon; however:
  Package bamfdaemon is not installed.
 dde-daemon depends on dde-polkit-agent; however:
  Package dde-polkit-agent is not installed.
 dde-daemon depends on xdotool; however:
  Package xdotool is not installed.
 dde-daemon depends on libfprint0; however:
  Package libfprint0 is not installed.
 dde-daemon depends on dnsmasq-base; however:
  Package dnsmasq-base is not installed.
 dde-daemon depends on cgroup-tools; however:
  Package cgroup-tools is not installed.
 dde-daemon depends on imwheel; however:
  Package imwheel is not installed.
 dde-daemon depends on procps; however:
  Package procps is not installed.
 dde-daemon depends on lastore-daemon; however:
  Package lastore-daemon is not installed.
 dde-daemon depends on deepin-installer-timezones; however:
  Package deepin-installer-timezones is not installed.
 dde-daemon depends on dde-api (>= 3.1.28); however:
  Package dde-api is not installed.

dpkg: error processing package dde-daemon (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 dde-daemon
[es02@greyhat ~]$ `

No good for non-deb :(

**Comment by electricface** [comment]
https://pan.baidu.com/s/14zSE2v25B4gByGYZy0amFQ
For users who are not deb systems, download the binary I shared.
 replace file /usr/lib/deepin-daemon/dde-session-daemon

**Comment by aiamuzz** [comment]
@electricface  ... a simple fix ... but a loooooong time to arrive at it ...

I have installed the said deb package ... i try to run this command to restart the dock ...

`killall dde-session-daemon; /usr/lib/deepin-daemon/dde-session-daemon`

previously the command would terminate to a new command line ... but it is no longer happening so ... the command seems to be stuck at ...

```
<info> manager_connection.go:171: add connection &network.connection{nmConn:(*networkmanager.ConnectionSettings)(0xc4203ad140), connType:"wireless", Path:"/org/freedesktop/NetworkManager/Settings/5", Uuid:"f026c281-a573-4dc9-8584-3cb4f7e02aa2", Id:"High5", HwAddress:"60:67:20:7E:14:E4", ClonedAddress:"", Ssid:"High5"}
<warning> dock_manager_xevent.go:154: WindowError{code=3, seq=191, major=2, minor=0, bad=29360924}

```

i close the terminal the dock is getting terminated ...

**Comment by electricface** [comment]
Dde-session-daemon is a daemon and does not exit immediately. I just fixed the panic problem.

**Comment by aiamuzz** [comment]
I think there is a conflict ... after i install the fix deb you've provided ... updates show up in the control center ... installing which seems to overwrite on the fix dev ... is the fix dev of a lesser version ?

```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
The following packages will be upgraded:
  dde-daemon
1 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/12.5 MB of archives.
After this operation, 2,048 B disk space will be freed.
Do you want to continue? [Y/n] y
(Reading database ... 392909 files and directories currently installed.)
Preparing to unpack .../dde-daemon_3.2.24-1_amd64.deb ...
Unpacking dde-daemon (3.2.24-1) over (3.2.24~gerrit+38055~1) ...
Setting up dde-daemon (3.2.24-1) ...
Processing triggers for dbus (1.12.8-2) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...

```

**Comment by aiamuzz** [comment]
The system is better after installing your deb file ... @electricface  ... but the experience is still lacking ... i can't switch to the full screen launcher ... app drawer ... stuck in Windows style launcher menu !!!

**Comment by BLumia** [comment]
Closing since it seems these issue no longer exist in 15.10 release. If you still got any issue, feel free to open another new issue with details to reproduce so we can help you out.
